test/integration/root_overview_test.rb

```diff
@@ -25,4 +25,282 @@ class RootOverviewTest < IntegrationTest
     assert_content("21 document(s)")
     assert_no_content("Guides and Gals")
   end
+
+  should "filter by assigned user" do
+    alice = FactoryBot.create(:user, :govuk_editor, name: "Alice", uid: "alice")
+    bob     = FactoryBot.create(:user, :govuk_editor, name: "Bob", uid: "bob")
+    charlie = FactoryBot.create(:user, :govuk_editor, name: "Charlie", uid: "charlie")
+
+    x = FactoryBot.create(:guide_edition, title: "XXX", slug: "xxx")
+    y = FactoryBot.create(:guide_edition, title: "YYY", slug: "yyy")
```

Comment:
may be we can name it bit better, even something like guide_edition_1, 2

Reply:
Makes sense, I've renamed them keeping the 'x' and 'y' - 'guide_edition_x', 'guide_edition_y'.